src-backend/api/serializer.py

```diff
@@ -44,11 +44,31 @@ def to_representation(self, instance):
         return ret
 
 
+class PageListSerializer(serializers.ListSerializer):
+
+    class Meta(object):
+        model = models.Page
+
+    def update(self, instance, validated_data):
+        current_page_mapping = {page.id: page for page in instance}
+        new_data_mapping = {item['id']: item for item in validated_data}
+
+        result = []
+        for page_id, data in new_data_mapping.items():
```

Comment:
It's pretty obvious that the page_id is from the new_data_mappings but whatever, renamed